src/main/resources/assets/waila/lang/zh_cn.json

```
@@ -76,5 +76,6 @@
   "config.waila.plugin_waila.show_entity_hp": "显示实体生命值",
   "config.waila.plugin_waila.show_states": "显示状态",
 
-  "entity.minecraft.villager.none": "傻子"
-}
\ No newline at end of file
+  "entity.minecraft.villager.none": "失业",
```

无业 is better?

---

没错，毕竟不是每一个人都被TNT炸了工作方块x
```suggestion
  "entity.minecraft.villager.none": "无业",
```